tools/analysis_tools/test_robustness.py

```diff
@@ -182,37 +91,76 @@ def parse_args():
     return args
 
 
+def trigger_visualization_hook(cfg, args):
```

Comment:
We can reuse this function in test.py by putting that from test.py to mmdet.engine.hooks.utils